client/java-armeria-legacy/src/main/java/com/linecorp/centraldogma/client/armeria/legacy/LegacyCentralDogma.java

```diff
@@ -326,7 +326,9 @@
                                                       String repositoryName,
                                                       Revision from,
                                                       Revision to,
-                                                      PathPattern pathPattern) {
+                                                      PathPattern pathPattern,
+                                                      int maxCommits) {
```

Comment:
nit; what do you think of explicitly throwing an exception if `maxCommits` is specified?

```
checkArgument(maxCommits == 0, "maxCommits is not supported in LegacyCentralDogma");
```

Reply:
Thanks, added. 😄